### PR TITLE
Fixed #4907 : [Android Wiki] "Get-Help" - Add link for Debugging Docs guide 

### LINKED ISSUE #4907: [Android Wiki] "Get-Help" - Add link for Debugging Docs guide
Add link of [debugging doc guide](https://github.com/oppia/oppia-android/wiki/Frequent-Errors-and-Solutions#how-to-write-a-debugging-doc) for oppia-android in [Get-Help](https://github.com/oppia/oppia-android/wiki/Get-Help#before-you-ask-a-general-question) wiki.

### PR DESCRIPTION
**Pull request by MohitGupta121**
## Explanation

Fixed #4907 : [Android Wiki] "Get-Help" - Add link for Debugging Docs guide 

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

### CODE REVIEW

File: wiki/Get-Help.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ paste a screenshot
 ### Before you ask a general question
 
 * We expect that you have already **set up Oppia on your machine**, and it is successfully running. (If not, kindly do that first!)
-* Prepare a debugging doc following [this template](https://docs.google.com/document/d/1OBAio60bchrNCpIrPBY2ResjeR11ekcN0w5kNJ0DHw8). For more information on how to use debugging docs, see [this link](https://github.com/oppia/oppia/wiki/Debugging-Docs).
+* Prepare a debugging doc following [this template](https://docs.google.com/document/d/1OBAio60bchrNCpIrPBY2ResjeR11ekcN0w5kNJ0DHw8). For more information on how to prepare a debugging doc following [the guidelines provided on the wiki page](https://github.com/oppia/oppia-android/wiki/Frequent-Errors-and-Solutions#how-to-write-a-debugging-doc).
```
Comment (seanlip) [inline wiki/Get-Help.md]: ...prepare a debugging doc, follow the guidelines provided on [this wiki page](...).

**Review by seanlip — CHANGES_REQUESTED** [review]

File: wiki/Get-Help.md (hunk 1 above)
Comment (MohitGupta121) [reply wiki/Get-Help.md]: Done updated.

**Review by seanlip — APPROVED** [review]

**Review by gp201 — APPROVED** [review]
LGTM

**Review by BenHenning — APPROVED** [review]
Thanks @MohitGupta121. This LGTM!

File: wiki/Get-Help.md (hunk 1 above)
Comment (BenHenning) [reply wiki/Get-Help.md]: This appears to be resolved in the latest changes, so marking it as resolved to unblock merging.

**Pull request merged**